Resources/Catalog/ConditionPlugins/FilenamePlugin.py

```diff
@@ -0,0 +1,59 @@
+"""
+  Defines the plugin to perform evaluation on the lfn name
+"""
+__RCSID__ = "$Id $"
+
+from DIRAC.Resources.Catalog.ConditionPlugins.FCConditionBasePlugin import FCConditionBasePlugin
+
+
+class FilenamePlugin( FCConditionBasePlugin ):
+  """
+     This plugin is to be used when filtering based on the LFN name
+  """
+
+  def __init__( self, conditions ):
+    """ the condition can be any method of the python string object that can be evaluated
+        as True or False:
+
+          * endswith
+          * find
+          * isalnum
+          * isalpha
+          * isdigit
+          * islower
+          * isspace
+          * istitle
+          * isupper
+          * startswith
+
+      It should be written just like if you were calling the python call yourself.
+      For example:
+        Filename=startswith('/lhcb')
+        Filename=istitle()
+
+    """
+    super( FilenamePlugin, self ).__init__( conditions )
+
+
+  def eval( self, **kwargs ):
```

Comment:
Kind of. They semanticaly do the same. And pylint does not complain about it. If it really is a problem I can change it though.